app/frontend/src/submissions/views/SubmissionsHome.vue

```diff
@@ -574,14 +576,121 @@ export default {
 
         // non-form fields that should be saved with form
         meta: {
-          drillerSameAsPersonResponsible: false
+          drillerSameAsPersonResponsible: false 
         }
       }
       this.componentUpdateTrigger = Date.now()
     },
+    groundwaterProtectionRegulationValidation(errors) {
+      const {
+        owner_full_name, 
+        owner_mailing_address,
+        owner_city,
+        owner_province_state,
+        owner_postal_code,
+      } = this.form
+
+      if (!owner_full_name) {
+        errors.owner_full_name = ['Owners Full Name Required.'];
+      }
+      if (!owner_mailing_address) {
+        errors.owner_mailing_address = ['Owners Mailing Address Required.'];
+      }
+      if (!owner_city) {
+        errors.owner_city = ['Owners City Required.'];
+      }
+      if (!owner_province_state) {
+        errors.owner_province_state = ['Owners Province or State Required.'];
+      }
+      if (!owner_postal_code) {
+        errors.owner_postal_code = ['Owners Postal Code Required.'];
+      }
+    },
+    validateWellIdentificationPlateFields(errors) {
+      const { 
+        well_class,
+        identification_plate_number,
+        well_identification_plate_attached,
+      } = this.form
+
+      let isWellIdentificationPlateToBeVerified = false;
+
+      if (well_class === WELL_CLASS.WATER_SUPPLY) { isWellIdentificationPlateToBeVerified = true; }
+      if (well_class === WELL_CLASS.INJECTION) { isWellIdentificationPlateToBeVerified = true; }
+      if (well_class === WELL_CLASS.RECHARGE) { isWellIdentificationPlateToBeVerified = true; }
+      
+      if (isWellIdentificationPlateToBeVerified == false) return;
+
+      if (!identification_plate_number) {
+        errors.identification_plate_number = ['Identification Plate Number Required.'];
+      }
+      if (!well_identification_plate_attached) {
+        errors.well_identification_plate_attached = ['Well Identification Plate Attached Required.'];
+      }
+    },
+    validateWellFields(errors) {
+      const { 
+        work_start_date, 
+        work_end_date, 
+        drilling_methods, 
+        total_depth_drilled, 
+        finished_well_depth
+      } = this.form;
+
+      if (!work_start_date) {
+        errors.work_start_date = ['Start Date of Work Required.'];
+      }
+      if (!work_end_date) {
+        errors.work_end_date = ['End Date of Work Required.'];
+      }
+
+      if (drilling_methods.length === 0) {
+        errors.drilling_methods = ['Drilling Methods Required.'];
+      }
+
+      if (!total_depth_drilled) { errors.total_depth_drilled = ['Total Depth Drilled Required.']; }
+      if (!finished_well_depth) { errors.finished_well_depth = ['Finished Well Depth Required.']; }
```

Comment:
Inconsistency in styling where `if`'s are single line vs multi-line, doing the same thing, in the same function